devicetypes/smartthings/zwave-switch-secure.src/zwave-switch-secure.groovy

```diff
@@ -22,9 +22,7 @@ metadata {
 
 		fingerprint inClusters: "0x25,0x98"
 		fingerprint deviceId: "0x10", inClusters: "0x98"
-		fingerprint mfr: "0086", prod: "0003", model: "008B", deviceJoinName: "Aeon Labs Nano Switch"
```

Comment:
@iTecHome those three fingerprints should stay in the DTH, please provide fix which will bring them back

Reply:
Done.